build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:4.0.1"
+        classpath 'com.android.tools.build:gradle:4.1.0'
```

Comment:
Is there a specific reason you switched to 4.1.0?

Reply:
Does the app run on your machine with this version number? My machine throws errors when using 4.1.0.

Reply:
That is the most recent version of gradle IIRC. Android Studio's been harassing me to upgrade but I've been too paranoid to do so

Reply:
Hm we should determine which version is best for our project. I was having problems with 4.1.0, but if @RealDogDad has found a way to make it work we should go with that.

Reply:
Oh interesting catch, I will look and see if the update breaks it.